app/components/finance/npq/payment_overviews/course.html.erb

```diff
@@ -34,13 +34,33 @@
         <td scope="row" class="govuk-table__cell govuk-body-s"><%= number_to_pounds calculator.output_payment_subtotal %></td>
       </tr>
 
+      <% if calculator.refundable_declarations_count.positive? %>
```

Comment:
The targeted delivery refundable declarations will be part of the clawbacks as well, is the intention to keep them in both  sections or should we subtract them from here?

Reply:
They shouldn't be subtracted. The target delivery declaration is an extra on the eligible declarations. You can't have clawback without "target delivery clawback"